src/app/system/manage-surveys/create-survey/create-survey.component.scss

```diff
@@ -0,0 +1,17 @@
+.delete-wrapper {
+  padding: 17.5px 0 0 0;
+}
+
+h2 {
+  font-weight: 500;
+  margin: 0;
+}
+
+h4 {
+  font-weight: 500;
+  margin: 0;
+}
+
+#margined-divider {
```

Comment:
It must be an ID, or else it would not override the classes applied by default by the inset.

Reply:
We should try to find a workaround for this, as the divider with the same id will be added multiple times depending on the number of questions. There should be only a single element with an id.